spec/requests/families_spec.rb

```diff
@@ -1,6 +1,59 @@
 require "rails_helper"
 
 RSpec.describe "Families", type: :request do
-  describe "GET /families" do
+  let(:partner) { create(:partner) }
```

Comment:
I tend to prefer creating records like this directly in the test over the usage of `let`. [This blog post](https://thoughtbot.com/blog/lets-not) is helpful in understanding why. This is just personal preference though, and wont be considered a blocker to merging.